template/README.md

```diff
@@ -287,7 +287,10 @@ Some editors, including Sublime Text, Atom, and Visual Studio Code, provide plug
 
 They are not required for linting. You should see the linter output right in your terminal as well as the browser console. However, if you prefer the lint results to appear right in your editor, there are some extra steps you can do.
 
-You would need to install an ESLint plugin for your editor first.  
+You would need to install an ESLint plugin for your editor first.
+
+>Note: if you are using the `linter-eslint` plugin on Atom, make sure that **Use global ESLint installation** option is checked.
```

Comment:
Let’s add a note with that format, it will be visible thanks to the screenshot.
Screenshot should be inline in the note.